sdk/messaging/azservicebus/receiver.go

```diff
@@ -309,21 +314,24 @@ func (r *Receiver) Close(ctx context.Context) error {
 }
 
 // CompleteMessage completes a message, deleting it from the queue or subscription.
+// This function can only be used when the Receiver has been opened with `ReceiveModePeekLock`.
```

Comment:
pkgsite doesn't support monospace formatting 😞 
```suggestion
// This function can only be used when the Receiver has been opened with ReceiveModePeekLock.
```